apps/studio/schemas/organizer.ts

```diff
@@ -0,0 +1,34 @@
+import { HeartIcon } from "@sanity/icons";
+import { defineField, defineType } from "sanity";
+
+export default defineType({
+  name: "organizer",
+  icon: HeartIcon,
+  title: "Organizers",
+  type: "document",
+  fields: [
+    defineField({
+      name: "name",
+      title: "Name",
+      type: "string",
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: "pronouns",
+      title: "Pronouns",
+      type: "string",
+    }),
+    defineField({
+      name: "linkedinURL",
+      title: "LinkedIn URL",
+      type: "url",
+      validation: (Rule) => Rule.required().uri({ scheme: ["http", "https"] }),
+    }),
+    defineField({
+      name: "profilePic",
+      title: "Profile Picture",
+      description: "Images should preferably have a 1:1 aspect ratio.",
+      type: "image",
+    }),
+  ],
```

Comment:
I think Sanity Studio is able to automatically assume we want to use this image for previews since it's the only one. Do you think it would be advantageous to explicitly state this in case this changes in the future?

Reply:
In that case, it'd be better to use the uploaded image for previews.

When you say "this", are you referring to the aspect ratio description?

Reply:
I'm referring to the image that Sanity uses in the list preview. For example, if you add an image to the document, it's used as the list preview image.
It might be worth adding the following to explicitly state that we want the name to be the title and profilePic to be the preview image.
```ts
preview: {
  select: {
    title: "name",
    media: "profilePic"
  }
}
```

Reply:
![Arc 2023-06-18 at 15 49 12@2x](https://github.com/HackAtUCI/hack.ics.uci.edu/assets/41758627/d3b77187-0cd7-4042-86af-d8a9ddd7fb2a)

Reply:
Yeah that would be helpful to include then, especially since this isn't like the URI validation where a developer can at least guess that `http` and `https` are required in order for the input value to be valid.

Reply:
Included in [`babb28d`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/babb28de063249dfcad0d491a4467125cb60e11d)